src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -339,6 +340,18 @@ web-ext run --start-url www.mozilla.com --start-url developer.mozilla.org
 
 Environment variable: `$WEB_EXT_START_URL`
 
+#### `--watch-ignored`
+A list of paths and globs patterns that should not be watched for changes. This is useful if you want to explicitly prevent web-ext from watching part of the extension directory tree, e.g. the node_modules folder.
+
+```shell
+web-ext run --watch-ignored path/to/file.js path/to/*.js path/**
```

Comment:
Nit, `path/to/*.js` wold be included in `path/**`, and `path/to/file.js` included in `path/to/*.js` and so the example is not realistic and may be confusing.

How about using 3 different name for the first level? e.g. something like:
- `dir1/to/file.js`
-  `dir2/*.js`
- `dir3/**`

Reply:
For sure!...

Reply:
Fixed.